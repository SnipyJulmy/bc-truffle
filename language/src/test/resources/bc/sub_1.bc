a = 321
b = 9328190
c = 20134315
d = 541321
e = 68462
a - b - c;
e - d - c;
a - c - b;
a - 1 - d;
-a - d - e;
d - e - a;
halt
