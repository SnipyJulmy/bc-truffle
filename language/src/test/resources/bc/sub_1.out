-29462184
-20607174
-29462184
-541001
-610104
472538
